### ISSUE TITLE
Upgrade to 0.8.7 from 0.8.4 causing crashes when used with iron:router

### ISSUE THREAD

**Issue by heschong** [kahmali/meteor-restivus#185]
The upgrade to 0.8.7 is causing crashes for any app using iron:router elsewhere in the application.

```
Error: Meteor code must always run within a Fiber. Try wrapping callbacks that you pass to non-Meteor libraries with Meteor.bindEnvironment.
    at Object.Meteor._nodeCodeMustBeInFiber (packages/meteor/dynamics_nodejs.js:9:1)
    at [object Object]._.extend.get (packages/meteor/dynamics_nodejs.js:21:1)
    at [object Object].RouteController.lookupOption (packages/iron_router/lib/route_controller.js:66:1)
    at new Controller.extend.constructor (packages/iron_router/lib/route_controller.js:26:1)
    at [object Object].ctor (packages/iron_core/lib/iron_core.js:88:1)
    at Function.Router.createController (packages/iron_router/lib/router.js:201:1)
    at Function.Router.dispatch (packages/iron_router/lib/router_server.js:39:1)
    at Object.router (packages/iron_router/lib/router.js:15:1)
    at next (/Users/redacted/.meteor/packages/webapp/.1.2.3.15kpnst++os+web.browser+web.cordova/npm/node_modules/connect/lib/proto.js:190:15)
    at Object.handle (/Users/redacted/.meteor/packages/simple_json-routes/.2.0.1.1a0fwo3++os+web.browser+web.cordova/npm/node_modules/connect-route/lib/connect-route.js:110:5)
```

There seems to be a conflict with iron:router and simple:json-routes maybe.  See also stubailo/meteor-rest#82

To reproduce:

```
meteor create foo
cd foo
meteor add nimble:restivus
meteor add iron:router
meteor run
```

**Comment by kahmali** [comment]
I think you're right. This appears to be a result of the upgrade to json-routes v2.0.1. For now, anyone using iron router should remain on Restivus v0.8.5 or below. 

Thanks so much for reporting this! The world will be a better place when everyone moves on to Flow Router. I'll try to update the docs with a warning or something.

**Comment by obiwarn** [comment]
can confirm

**Comment by heschong** [comment]
simple:routes@2.1.0 fixes this

**Comment by kahmali** [comment]
Nice! I'll update ASAP and release a new version. Thanks for the heads up @heschong!

**Comment by kahmali** [comment]
Just published v0.8.8 with the updated version of `simple:json-routes`. Thanks again for the heads up @heschong. Let me know if any of you are still getting errors.
